Index: README.md
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># Introduction to Git and GitHub\r\n\r\n## Simple Interest Calculator\r\n\r\nA calculator that calculates simple interest given principal, annual rate of interest and time period in years.\r\n\r\n```\r\nInput:\r\n   p, principal amount\r\n   t, time period in years\r\n   r, annual rate of interest\r\nOutput\r\n   simple interest = p*t*r\r\n```\r\n\r\n\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/README.md b/README.md
--- a/README.md	(revision 29c51f979b2c5e117d54210e2c459c7ac1218a53)
+++ b/README.md	(date 1701115603731)
@@ -12,5 +12,5 @@
 Output
    simple interest = p*t*r
 ```
-
+2023 XYZ, Inc.
 
